frontend/src/views/components/Dashboard/GraphGrid/GraphGrid.tsx

```diff
@@ -1,29 +1,37 @@
 import Grid from "@mui/material/Grid2";
-import { Container, Card, CardContent, CardHeader } from "@mui/material";
+import { Container, Card, CardContent, CardHeader, Typography } from "@mui/material";
+import { ResponsiveBar } from "@nivo/bar";
+import { colorSchemes } from '@nivo/colors';
 import BarChart from "../BarChart/BarChart";
 
 function Graph({data, title}) {
     return (
-        <Card p={3} style={{ width: "325px" }}>
-            <CardHeader title={title}/>
```

Comment:
also extremelyy nitpicky but i feel like the spacing between the title and the graph of the cards is too big, looks a little off but is a small deal